mitodl/mitxonline
# Remove new-design styles | Cleaning up duplicated styles

### What are the relevant tickets?
Clean up

### Description (What does it do?)
Removes the `.new-design` class and removes duplicated styles.

### Screenshots (if appropriate):
Everything should look as before

### How can this be tested?
Go to home page, product page and other pages to make sure that everything looks normal.
